new  qc json for trd

bazinski opened the pull request.

knopers8 commented on MC/config/QC/json/trd-digits-task.json:
```diff
@@ -9,70 +9,139 @@
+          "pulseheightpeakupper": "5.0"
+        },
+        "location": "remote",
+	"saveObjectsToFile":"qcrootobjects.root"
```
Please disable saving objects to a file (remove this line), so QC on grid does not generate more files than necessary. The same comment applies to other tasks.

bazinski replied:
done

knopers8 commented on MC/config/QC/json/trd-digits-task.json:
```diff
@@ -9,70 +9,139 @@
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD;rawstats:TRD/RAWSTATS"
```
Are you sure that `rawstats:TRD/RAWSTATS` is accesible at this stage of computation (or in simulated data at all)?
Here is what TRD QC requests to be present:
https://github.com/AliceO2Group/O2DPG/blob/ca1971d3835f9f5186f488993281e3960fecdd95/MC/bin/o2dpg_sim_workflow.py#L785
Here is the documentation about adding QC to MC scripts:
https://github.com/AliceO2Group/O2DPG/tree/master/MC#adding-qc-tasks-to-the-simulation-script

bazinski replied:
rawstats would not work no, and also the tracklet task is removed for not wanting to edit the python.

knopers8 commented on MC/config/QC/json/trd-digits-task.json:
```diff
@@ -9,70 +9,139 @@
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
```
you can delete the line in such case.

bazinski replied:
done

knopers8 commented on MC/config/QC/json/trd-digits-task.json:
```diff
@@ -9,70 +9,139 @@
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        },
+        "location": "remote",
```
this is not applicable in the case of batch processing, you can delete all "location" lines.

knopers8 reviewed (changes requested):
Thanks for the update. I see that you added some new tasks, which in principle should be fine, since the tasks request the same input to be present at this stage. This is not entirely true for the `RawDataTask`, see the corresponding comment.
Additionally, please consider applying also the other suggestions.

bazinski commented:
Hi Piotr, ja its clear I just took standard qc file and replaced the old non functioning one and switched off the rawdata task which receives a stats message directly from the raw data reader, and hence not relevant to MC.  
Will fix now, and disable pulsheight task which i think is irrelevant for MC.

knopers8 commented:
Hi @bazinski, could you please apply the changes? Unless you are still on holidays, then sorry for bothering you.

bazinski commented:
HI @knopers8 oops this completely fell off the todo list. 
I removed the tracklet qc after I looked at the python and did not want to touch, maybe later.
All your comments done.

bazinski commented on MC/config/QC/json/trd-digits-task.json:
```diff
@@ -9,70 +9,139 @@
       "consul": {
-        "url": "http://localhost:8500"
+        "url": "http://consul-test.cern.ch:8500"
       },
```
done

knopers8 reviewed (approved):
Thank you!
